### PR TITLE
Stop panics caused by dereference in runc

### PR DESCRIPTION
**Pull request by jwhonce**
https://bugzilla.redhat.com/show_bug.cgi?id=1718441

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
@giuseppe PTAL

File: libcontainer/generic_error.go [hunk 1]
```diff
@@ -93,6 +93,10 @@ func (e *genericError) Error() string {
 	if e.Cause == "" {
 		return e.Message
 	}
+
+	if e.Stack.Frames == nil {
```
Comment (rhatdan) [inline libcontainer/generic_error.go]: Should this be len(e.Stack.Frames=0)?

Comment (TomSweeneyRedHat) [reply libcontainer/generic_error.go]: or maybe an 'or'?  I'm not sure.

```
if e.Stack.Frames == nil || len(e.Stack.Frames=0) {
```

Comment (jwhonce) [reply libcontainer/generic_error.go]: Couldn't hurt to add the extra check.

**Comment by rhatdan** [comment]
LGTM  Is this check in the upstream?

**Comment by giuseppe** [comment]
LGTM, but as @rhatdan pointed out.  We should fix it upstream as well if it is affected

**Comment by TomSweeneyRedHat** [comment]
Just curious, looks like you were checking both len and nil, then repushed with just a len check.  Did you find the nil check was superfluous?

**Comment by jwhonce** [comment]
@TomSweeneyRedHat Yes, len() does a nil or zero check.

**Comment by jwhonce** [comment]
@giuseppe I was waiting on an upstream PR until this one was merged/okayed.

**Pull request merged**